### PR TITLE
chore: prep for minting components

### PR DESCRIPTION
**Pull request by DavidMinarsch**
Missing:
- [x] dependencies
- [x] prep metadata
- [x] mint components

### CODE REVIEW

File: mints/mapping.txt [hunk 1]
```diff
@@ -0,0 +1,15 @@
+token_id-"component_id"
+X-"protocol/valory/default/1.0.0"
+X-"protocol/valory/acn_data_share/0.1.0"
+X-"connection/valory/websocket_client/0.1.0" -> "protocol/valory/default/1.0.0"
+WARNING: might we want to call the associated protocol websocket? For forward compatibility to when we change it
```
Comment (DavidMinarsch) [inline mints/mapping.txt:5]: @0xArdi I know this would create extra work. Thoughts?

File: mints/mapping.txt [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+X-"contract/valory/agent_mech/0.1.0"
+X-"skill/valory/contract_subscription/0.1.0" -> "protocol/valory/default/1.0.0", "connection/valory/websocket_client/0.1.0"
+X-"skill/valory/multiplexer_abci/0.1.0" -> 3, 14, "contract/valory/agent_mech/0.1.0"
+X-"skill/valory/task_execution_abci/0.1.0" -> 3, 11, 19, 20, "contract/valory/agent_mech/0.1.0", "protocol/valory/acn_data_share/0.1.0", 14, 22
+X-
+What happened to transaction preparation?
```
Comment (DavidMinarsch) [inline mints/mapping.txt:11]: @Adamantios & @0xArdi

Comment (Adamantios) [reply mints/mapping.txt:11]: Not sure, it was removed on b6d31612c003e2c5a23cfa1a76f0e6f85246041e.

Comment (0xArdi) [reply mints/mapping.txt:11]: It only complicated things, especially when we added the multisend.

File: mints/mapping.txt (hunk 1 above)
Comment (0xArdi) [reply mints/mapping.txt:5]: In reality it is the default protocol, is there any benefit to renaming it?

Comment (DavidMinarsch) [reply mints/mapping.txt:5]: It's fine. If we knew we'd immediately replace with websocket protocol then yes there'd be a benefit

**Pull request merged**